src/schema/query_options.js

```diff
@@ -193,7 +194,7 @@ const custom_filters = [
   "dependson",
   "whiteboard",
   "creation_ts",
-  "severity",
+  "type",
```

Comment:
I don't think we are sorting by type, with severity removed we should remove "bug_severity" from sort_keys below